oio/xcute/common/backend.py

```diff
@@ -537,7 +537,7 @@ def list_jobs(self, marker=None, limit=1000):
 
             pipeline = self.conn.pipeline()
             for job_id in job_ids:
-                self._get_job_info(job_id, client=pipeline)
+                self._get_job_info(job_id.decode(), client=pipeline)
```

Comment:
For clarity, the `encoding` parameter should be specified. Like here : https://github.com/open-io/oio-sds/blob/afefff8689b1e1fed3f880dbe5904b0131b22e41/oio/xcute/common/backend.py#L704

Reply:
:+1: